src/Application/Cli/TravisAutoDeployCommand.php

```diff
@@ -58,13 +58,6 @@ protected function configure()
                 getcwd()
             )
             ->addOption(
-                'php-version',
-                null,
-                InputOption::VALUE_REQUIRED,
-                'Specify for which php version you want to deploy documentation, mainly to avoid multiple deploys',
-                '5.4'
-            )
```

Comment:
I updated the travis configuration to avoid this option, which were only there as a little hack to avoid re-doing the same action on each php version... So not needed anymore I think... But could require some documentation for sure...